windows-vm-configure-winrm/azuredeploy.json

```diff
@@ -0,0 +1,244 @@
+                                "id": "[variables('subnetRef')]"
+                            }
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "apiVersion": "2015-05-01-preview",
+            "type": "Microsoft.Compute/virtualMachines",
+            "name": "[variables('vmName')]",
+            "location": "[resourceGroup().location]",
+            "dependsOn": [
+                "[concat('Microsoft.Storage/storageAccounts/', parameters('newStorageAccountName'))]",
+                "[concat('Microsoft.Network/networkInterfaces/', variables('nicName'))]"
+            ],
+            "properties": {
+                "hardwareProfile": {
+                    "vmSize": "[variables('vmSize')]"
+                },
+                "osProfile": {
+                    "computername": "[variables('vmName')]",
+                    "adminUsername": "[parameters('adminUsername')]",
+                    "adminPassword": "[parameters('adminPassword')]"
+                },
+                "storageProfile": {
+                    "imageReference": {
+                        "publisher": "[variables('imagePublisher')]",
+                        "offer": "[variables('imageOffer')]",
+                        "sku" : "[parameters('windowsOSVersion')]",
+                        "version":"latest"
+                    },
+                   "osDisk" : {
+                        "name": "osdisk",
+                        "vhd": {
+                            "uri": "[concat('http://',parameters('newStorageAccountName'),'.blob.core.windows.net/',variables('vmStorageAccountContainerName'),'/',variables('OSDiskName'),'.vhd')]"
+                        },
+                        "caching": "ReadWrite",
+                        "createOption": "FromImage"
+                    }
+                },
+                "networkProfile": {
+                    "networkInterfaces": [
+                        {
+                            "id": "[resourceId('Microsoft.Network/networkInterfaces',variables('nicName'))]"
+                        }
+                    ]
+                }
+            },
+            "resources": [
+              {
+                  "type": "Microsoft.Compute/virtualMachines/extensions",
+                  "name": "[concat(variables('vmName'),'/CustomScriptExtension')]",
+                  "apiVersion": "2015-05-01-preview",
+                  "location": "[resourceGroup().location]",
+                  "dependsOn": [
+                      "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'))]"
+                  ],
+                  "properties": {
+       	           "publisher": "Microsoft.Compute",
+       	           "type": "CustomScriptExtension",
+                     "typeHandlerVersion": "1.2",
+                     "settings": {
+                         "fileUris": [ "https://azurergtaskstorage.blob.core.windows.net/winrm/ConfigureWinRM.ps1" ],
```

Comment:
Is there any way you can place this file into this pull request, rather than an azure blob so folks know what it does? And be sure to follow the [raw.githubusercontent.com links guidance](https://github.com/azure/azure-quickstart-templates#rawgithubusercontentcom-links)